torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -1247,6 +1368,22 @@ def _init_local_optimizer(self) -> None:
                 "functional optimizer with more than one parameter group"
             params = param_groups[0]["params"]
             self.optim: Any = self._optim_constructor(params, **self._optim_defaults)
+
+            # Log information about the DDP and ZeRO bucketing
+            if logging.root.level >= logging.INFO:
```

Comment:
This information is helpful for experiments. It does not necessarily need to be included into `master` if not desired.

Reply:
maybe we should reuse `DistributedDebugLevel`?

https://github.com/pytorch/pytorch/blob/a016150163a7a78091bd965050f311d8af870abf/torch/csrc/distributed/c10d/Utils.hpp#L36

Reply:
Changed `if logging.root.level >= logging.INFO` to `if dist._get_debug_mode() != dist._DistributedDebugLevel.OFF` to follow:
https://github.com/pytorch/pytorch/blob/44fad84bca11a34b58d79e85ab81d28ac9cd80a8/torch/csrc/distributed/c10d/reducer.cpp#L111-L115